.github/workflows/deploy-javascore-testnet.yml

```diff
@@ -0,0 +1,237 @@
+                --step_limit=3000000000 \
+                --uri $GOLOOP_RPC_URI \
+                --key_store=./key.json | tee | jq -r . > tx.icon.bts
+
+      - name: Grab txresult
+        working-directory: ./javascore
+        shell: bash
+        timeout-minutes: 1
+        env:
+          GOLOOP_RPC_URI: ${{ secrets.GOLOOP_RPC_URI }}
+        run: |
+          while true; do
+            str=`goloop rpc --uri $GOLOOP_RPC_URI txresult $(cat tx.icon.bts) | tee tx-icon-bts || exit 0`
+            echo Output: $str
+            if cat tx-icon-bts | jq ".status" | grep -q "0x1"; then
+              break
+            fi
+            echo Wait
+            sleep 2
+          done
+          echo Finished
+
+      - name: Get tx address
+        run: |
+          cat ./javascore/tx-icon-bts | jq -r .scoreAddress > ./javascore/icon-addr-bts
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: icon-addr-bts
+          path: |
+            ./javascore/icon-addr-bts
+
+  register-bts-to-bmc:
+    needs: [deploy-bmc-javascore, deploy-bts-javascore]
+    name: register bts contract in bmc on testnet
+    runs-on: ubuntu-latest
+    container:
+      image: iconbridge/build
+      options: --user 1001
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Add SHORT_SHA env property with commit short sha
+        run: echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-7`" >> $GITHUB_ENV
+
+      - name: Retrieve the secret and decode it to a file
+        working-directory: ./javascore
+        env:
+          GOLOOP_RPC_KEY_STORE_B64: ${{ secrets.GOLOOP_RPC_KEY_STORE_B64 }}
+        run: |
+          echo $GOLOOP_RPC_KEY_STORE_B64 | base64 -d > key.json
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: icon-addr-bmc
+          path: |
+            ./javascore/
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: icon-addr-bts
+          path: |
+            ./javascore/
+
+      - name: Register BTS with BMC
+        working-directory: ./javascore
+        shell: bash
+        timeout-minutes: 1
+        env:
+          GOLOOP_RPC_KEY_PASSWORD: ${{ secrets.GOLOOP_RPC_KEY_PASSWORD }}
+          GOLOOP_RPC_URI: ${{ secrets.GOLOOP_RPC_URI }}
+        run: |
+          goloop rpc sendtx call --to $(cat icon-addr-bmc) \
+                --method addService \
+                --value 0 \
+                --param _addr=$(cat icon-addr-bts) \
+                --param _svc="bts" \
+                --step_limit=3000000000 \
+                --uri $GOLOOP_RPC_URI \
+                --key_store=./key.json \
+                --nid=0x2 | tee | jq -r . > tx.icon.bts.bmc
+      - name: Grab txresult
+        working-directory: ./javascore
+        shell: bash
+        timeout-minutes: 1
+        env:
+          GOLOOP_RPC_URI: ${{ secrets.GOLOOP_RPC_URI }}
+        run: |
+          while true; do
```

Comment:
the same

Reply:
There's timeout of 1 minute, so this case will be handled to go infinite loop.